deployer/templates/es.yaml

```diff
@@ -41,6 +41,7 @@ objects:
             component: ${ES_CLUSTER_NAME}
             deployment: ${ES_DEPLOYMENT_NAME}
         spec:
+          terminationGracePeriodSeconds: 1800
```

Comment:
We can drop it down to 10 minutes. I was planning for a worst case where they have a ton of indices that the metadata needed to be written out for and they could always oc delete the pod if its taking too long. Since we don't have actual performance number on this it was kind of a guess